packages/discovery-provider/Dockerfile.prod

```diff
@@ -88,7 +88,4 @@ ENV audius_loggly_tags=$audius_loggly_tags
 
 EXPOSE 5000
 
-HEALTHCHECK --interval=5s --timeout=10s --retries=12 \
```

Comment:
moved to compose 
https://github.com/AudiusProject/audius-docker-compose/pull/474